sql-statements/sql-statement-show-config.md

```diff
@@ -7,6 +7,10 @@ summary: Overview of the use of SHOW CONFIG in the TiDB database
 
 The `SHOW CONFIG` statement is used to show the current configuration of various components of TiDB. Note that the configuration and system variables act on different dimensions and should not be mixed up. If you want to obtain the system variable information, use the [SHOW VARIABLES](/sql-statements/sql-statement-show-variables.md) syntax.
 
+> **Note:**
+>
+> This feature is not available on [TiDB Serverless](https://docs.pingcap.com/tidbcloud/select-cluster-tier#tidb-serverless) clusters.
+
```

Comment:
It is not available on both serverless and dedicated.

Reply:
There might be some other features are not available on both.

Reply:
Thanks for the catch. I will double check the support info for other features again.